modules/luci-mod-status/htdocs/luci-static/resources/view/status/channel_analysis.js

```diff
@@ -314,6 +314,22 @@ return view.extend({
 					if (res.vht_operation.channel_width == 80) {
 						chan_width = 8;
 						res.channel_width = "80 MHz";
+
+						/* If needed, adjust based on the newer interop workaround. */
+						if (res.vht_operation.center_freq_2) {
+							var diff = res.vht_operation.center_freq_2 -
+								res.vht_operation.center_freq_1;
+							diff = diff < 0 ? -diff: diff;
```

Comment:
Since you wrote that this is your first JavaScript contribution, you may not know about the [Math.abs function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs).

```suggestion
							diff = Math.abs(diff);
```

Reply:
Fixed :D